service/file_transaction_service.py

```diff
@@ -86,9 +88,22 @@ def CreateUserFolder(self, request, context):
         is_uuid_valid = utility.verify_uuid(request.uuid)
 
         if is_uuid_valid:
-            count = utility.count_folders(request.uuid)
-            logger.info("count:", count)
-            created = utility.create_uuid_container_in_azure(count, request.uuid)
+            # Make Lock for this uuid if it doesn't exist already
+            if request.uuid not in self.__server.get_uuid_locker():
+                self.__server.get_uuid_locker()[request.uuid] = Lock()
+
+            # Lock this uuid
+            self.__server.get_uuid_locker()[request.uuid].acquire()
+
+            try:
+                count = azure_client.count_folders_in_azure(request.uuid)
+                logger.info("count:", str(count))
+                created = azure_client.create_uuid_container_in_azure(count, request.uuid)
+            except:
```

Comment:
Pep 8 style guide error
Consult https://stackoverflow.com/questions/30442236/how-to-prevent-too-broad-exception-in-this-case